mobile/src/bundles/common/components/modal/modal.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { Modal as RNModal } from 'react-native';
+
+import {
+    Button,
+    Text,
+    TouchableOpacity,
+    View,
+} from '~/bundles/common/components/components';
+import { ButtonType, TextCategory } from '~/bundles/common/enums/enums';
+import { globalStyles } from '~/bundles/common/styles/styles';
+
+import { styles } from './styles';
+
+type Properties = {
+    title: string;
+    visible: boolean;
+    acceptText?: string;
+    children: React.ReactNode;
+    onClose: () => void;
+    onAccept?: () => void;
+};
+
+const Modal: React.FC<Properties> = ({
+    title,
+    visible,
+    acceptText = 'Accept',
+    children,
+    onClose,
+    onAccept,
+}) => {
+    return (
+        <RNModal
+            animationType="fade"
+            transparent
+            visible={visible}
+            onRequestClose={onClose}
+        >
+            <TouchableOpacity
+                style={[
+                    styles.backdrop,
+                    globalStyles.flex1,
+                    globalStyles.justifyContentCenter,
+                    globalStyles.alignItemsCenter,
+                ]}
+                onPress={onClose}
+            >
+                <View
+                    style={[
+                        styles.container,
+                        globalStyles.p25,
+                        globalStyles.borderRadius10,
+                    ]}
+                >
+                    <Text category={TextCategory.H4}>{title}</Text>
+                    <View style={styles.content}>{children}</View>
+                    <View
+                        style={[
+                            globalStyles.flexDirectionRow,
+                            globalStyles.justifyContentFlexEnd,
+                        ]}
+                    >
+                        <Button
+                            buttonType={ButtonType.GHOST}
+                            label="Cancel"
+                            onPress={onClose}
+                        />
+                        {onAccept && (
+                            <Button
+                                buttonType={ButtonType.GHOST}
+                                label={acceptText}
+                                onPress={onAccept}
+                            />
+                        )}
+                    </View>
+                </View>
+            </TouchableOpacity>
+        </RNModal>
+    );
+};
```

Comment:
One of usages is choosing image picker source, think it's better just receive children inside with logic from parent. Need Katerina opinion.

Reply:
To be honest I do not fully understand what I should do. Should I create one more component which will just accept only children and no other props?

Reply:
Yes, because we need something like this in future and that Modal with buttons is also needed for us 
<img width="403" alt="image" src="https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/assets/103518438/84f6021f-859f-4169-a97d-e2e76185b222">

Reply:
I've created simple modal which accepts 3 props (children, onClose (to close the modal by clicking outside), and visible)